### PR TITLE
Import remaining issues for 2018

### PR DESCRIPTION
**Pull request by dellsystem**
For issue #35 (halfway done). Some things I noticed while doing the import:

* Some newsletters didn't have a "Worker's Perspective" header at all
* Sometimes the order of excerpt/image/header doesn't quite match our template (but that's fine, I just re-ordered)
* There's usually an <hr> between sections, which doesn't get caught by the script - I put them in manually, but probably missed a bunch

Also, I didn't get all of the embedded images (I was just lazy).

### CODE REVIEW

**Comment by jessesquires** [comment]
lol wow Tom Apple is displeased with the prose.

**Comment by dellsystem** [comment]
I can't run ImageOptim as I'm on Linux so I used pngquant instead. I shrunk some images from the previous import too (issues 17 and 18). @jessesquires does this look okay now?

**Comment by jessesquires** [comment]
@dellsystem looks much better! i'll run ImageOptim real quick, and see if it compresses any further, and push a commit here.

**Comment by jessesquires** [comment]
@dellsystem Oh -- I just realized some issues have multiple images, not just the header image.

We have an include for that.

You can use:

```python
{% include image.html
    file='file.jpg'
    alt='alt text'
    caption='caption text'
    source_link='link'
    half_width=true
%}
```

Then these other images will get the same styles, etc. as the header images.

**Comment by dellsystem** [comment]
oh amazing, i'll fix that now, thanks @jessesquires

**Comment by jessesquires** [comment]
Pushed that commit after running ImageOptim -- I have no idea what that thing does (and how there's so much extra crap in image files?!), but it's amazing. 😄

**Comment by dellsystem** [comment]
all the old posts use {% include image.html %} now!

File: _posts/2018-06-15-issue-4.md [hunk 1]
```diff
@@ -90,7 +98,7 @@ _All that harms labor is treason to America. No line can be drawn between these
 _I am glad to see that a system of labor prevails under which laborers can strike when they want to ... I like the system which lets man quit when he wants to and wish it might prevail everywhere._
   
 _The stronger bond of human sympathy, outside of the family relation, should be one uniting all working people of all nations, tongues and kindreds._  
-    
-- Abraham Lincoln
+
+―Abraham Lincoln
```
Comment (jessesquires) [inline _posts/2018-06-15-issue-4.md:102]: lol we quoted Lincoln?

Comment (dellsystem) [reply _posts/2018-06-15-issue-4.md:102]: :shrug: lol

**Comment by jessesquires** [comment]
@dellsystem looks awesome! I just pushed a commit after running ImageOptim one more time.

**Comment by dellsystem** [comment]
yay, thanks @jessesquires! i'll merge now and get to work on 2019 :skull:

**Comment by dellsystem** [comment]
do you know why Danger isn't succeeding, btw?

**Comment by dellsystem** [comment]
i can't merge while that's failing :cry: seems like some sort of unicode error. maybe it's some unicode character i added to an .md file

**Comment by jessesquires** [comment]
> i'll merge now and get to work on 2019 💀

oh shit. I didn't realize there was more! 🤣 

> i can't merge while that's failing 😢 seems like some sort of unicode error. maybe it's some unicode character i added to an .md file

Ah, that's probably it. 

Let me make this temporarily not required and I'll merge, then turn it back on 😄

**Pull request merged**